contrib/util/restore

```diff
@@ -154,6 +158,21 @@ while [ $LOOKING -eq 1 ]; do
     dlg_info "Extracting $TARFILE ..."
     cd $BAKDIR
     tar -xf $TARFILE
+    #RP_ADDED to get names of OEDIR ans webroot name if different from default by reading values from openemr_setup.txt
+    filename=$BAKDIR/openemr_setup.txt
```

Comment:
I think you should allow for the case where openemr_setup.txt does not exist, which will be the case for old backups and perhaps also backups created with a home-grown method.